server/auth/gatekeeper.go

```diff
@@ -337,11 +342,13 @@ func addClaimsLogFields(claims *types.Claims, fields log.Fields) log.Fields {
 	return fields
 }
 
-func DefaultClientForAuthorization(authorization string) (*rest.Config, *servertypes.Clients, error) {
+func DefaultClientForAuthorization(authorization string, qps float32, burst int) (*rest.Config, *servertypes.Clients, error) {
```

Comment:
this is the "default" client, so I don't think it should be configurable, by definition

Reply:
oh I see, this won't type-check otherwise (or necessarily work properly).... hmmm... I feel like we shouldn't have two `restConfig`s, which would simplify this. we can merge them, although that doesn't quite fix the type-check issue here... I wonder if the merge should be done in `server.go` instead... 🤔

Reply:
at minimum, I think it would make sense for this to accept a `rest.Config` instead of `qps` and `burst` in isolation. then we just pass through `s.restConfig`.

whether that actually makes sense for a "default" to have a configuration could be refactored later, I suppose. I need to look at the initialization chain a bit more

Reply:
ok it seems like the `clientForAuthorization` is only exposed for testing purposes, otherwise the default is always used, so yea I guess we'd merge `restConfig`s here

Reply:
@shuangkun you might've misunderstood what I said here (that's partly my fault, I had some scattered thoughts) -- I think it would make more sense to accept a `rest.Config` instead of a `qps` and `burst` in isolation

Reply:
Oh, do you mean this
```
type ClientForAuthorization func(authorization string, config *rest.Config) (*rest.Config, *servertypes.Clients, error)
```
I have change it.

Reply:
Yea that's what I meant, although I also meant merging the entirety of the `rest.Config` as well, see below comment